docs/keyboard-shortcuts.md

```diff
@@ -0,0 +1,59 @@
+| -------- | ---------------------------------------------------------------------------------------------------------------------- |
+| `Ctrl+C` | Exit the application. Press twice to confirm. If the input prompt is not empty, the first press will clear the prompt. |
+| `Ctrl+D` | Exit the application if the input is empty. Press twice to confirm.                                                    |
+| `Ctrl+L` | Clear the screen.                                                                                                      |
+| `Ctrl+O` | Toggle the display of the debug console.                                                                               |
+| `Ctrl+T` | Toggle the display of tool descriptions.                                                                               |
+| `Ctrl+S` | Allows long responses to print fully, disabling truncation. Use your terminal's scrollback to view the entire output.  |
+| `Esc`    | Close dialogs and suggestions.                                                                                         |
+| `Ctrl+Y` | Toggle auto-approval (YOLO mode) for all tool calls.                                                                   |
+
+## Input Prompt
+
+| Shortcut                                           | Description                                                                                                                         |
+| -------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------- |
+| `Enter` / `Ctrl+Enter`                             | Submit the current prompt.                                                                                                          |
+| `\` (at end of line) + `Enter`                     | Insert a newline.                                                                                                                   |
+| `Up Arrow`                                         | Navigate up through the input history.                                                                                              |
+| `Down Arrow`                                       | Navigate down through the input history.                                                                                            |
+| `Ctrl+P`                                           | Navigate up through the input history.                                                                                              |
+| `Ctrl+N`                                           | Navigate down through the input history.                                                                                            |
+| `Tab`                                              | Autocomplete the current suggestion.                                                                                                |
```

Comment:
this is a bit misleading and perhaps duplicative as you already show that tab accepts when their is a suggestion list. Consider just removing this or calling out that it accepts the current autocomplete suggestion if there is one.